Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/SDK/Maya/Scripts/Python/legacy_asset_converter/main.py

```diff
@@ -578,7 +580,8 @@ def initialize_window(self):
         self.initialize_qprocess()
 
         # Get Material Definition Template
-        with open(self.default_material_definition) as json_file:
+        mat_abs_path = Path(__file__).with_name(self.default_material_definition)
+        with open(mat_abs_path) as json_file:
```

Comment:
Can use pathlib.Path's open
```suggestion
        with mat_abs_path.open() as json_file:
```